common/app/common/dfp/TagSponsorship.scala

```diff
@@ -77,13 +92,38 @@ object HighMerchandisingTargetedTagSet {
   }
 }
 
+
 case class HighMerchandisingTargetedTagSet(items: Set[String] = Set.empty){
 
   def hasTag (tag: Tag): Boolean = items.exists(item => tag.id.endsWith(item))
 
+  def hasAdUnit (adUnitSuffix : String): Boolean = {
+    println(adUnitSuffix)
+    true
+  }
+
 
   def nonEmpty = items.nonEmpty
+}
+
+case class HighMerchandisingLineItemTargetsSeq(lineItems : Seq[HighMerchandisingLineItem] = Seq.empty) {
+
+  def hasAdUnitAndTag (adUnitSuffix: String, tags:Seq[Tag]): Boolean = {
+
+    val tagSeq = tags.map(tag => tag.name)
+    val tagSeqModify = tagSeq.map(oldTag => oldTag.replaceAll(" ","-").toLowerCase)
```

Comment:
Could do these two steps in one:
`tags.map(_.name.replaceAll(" ","-").toLowerCase)`